client/post-editor/editor-term-selector/index.jsx

```diff
@@ -50,23 +54,29 @@ class EditorTermSelector extends Component {
 	}
 
 	render() {
-		const { siteId, taxonomyName } = this.props;
+		const { postType, siteId, taxonomyName } = this.props;
 
 		return (
-			<TermTreeSelector
-				analyticsPrefix="Editor"
-				onChange={ this.boundOnTermsChange }
-				selected={ this.getSelectedTermIds() }
-				taxonomy={ taxonomyName }
-				siteId={ siteId }
-				multiple={ true }
-			/>
+			<div>
+				<TermTreeSelector
+					analyticsPrefix="Editor"
+					onChange={ this.boundOnTermsChange }
+					selected={ this.getSelectedTermIds() }
+					taxonomy={ taxonomyName }
+					siteId={ siteId }
+					multiple={ true }
+				/>
+				<AddTerm taxonomy={ taxonomyName } postType={ postType } />
```

Comment:
Do we need `postType` to be passed from `<EditorTermSelector />`, or couldn't `<AddTerm />` do just the same lookup in its `connect`?